File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrCustomer_Search.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Data;
using NotificationSystem.NotificationSystem.Data.NotificationSystemTableAdapters;
using static NotificationSystem.NotificationSystem.Data.NotificationSystem;
using NotificationSystem.NotificationSystem.Data.Classes;
using System.Diagnostics;
using System.Globalization;
using System.IO;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Security;
using System.Text;
using System.Threading.Tasks;
using Microsoft.VisualBasic;

namespace NotificationSystem.NotificationSystem.Web
{
    public partial class ctrCustomer_Search : System.Web.UI.UserControl
    {
        private int m_CustomerID = 0;

        public int CustomerID
        {
            get
            {
                return m_CustomerID;
            }
            set
            {
                m_CustomerID = value;
            }
        }
        protected void Page_Load(object sender, EventArgs e)
        {

        }

        public void PopulateSearchControl()
        {
            m_CustomerID = int.Parse(this.txtcustomerid.Text);
        }
    }
}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrCustomer_Search.ascx.vb
Imports System
Imports System.Web.UI

Public Partial Class ctrCustomer_Search
    Inherits UserControl
    Public m_CustomerID As Integer = 0

    Public Property CustomerID As Integer
        Get
            Return m_CustomerID
        End Get
        Set(value As Integer)
            m_CustomerID = value
        End Set
    End Property
    Protected Sub Page_Load(sender As Object, e As EventArgs)

    End Sub

    Public Sub PopulateSearchControl()
        m_CustomerID = Integer.Parse(txtcustomerid.Text)
    End Sub
End Class
